src/main/scala/util/RunUtils.scala

```diff
@@ -336,6 +453,8 @@ object IRTransform {
 /** Methods relating to program static analysis.
   */
 object StaticAnalysis {
+  var memoryRegionAnalysisResults: Map[CfgNode, LiftedElement[Set[MemoryRegion]]] = Map()
+  var mmmResults: MemoryModelMap = MemoryModelMap()
```

Comment:
We don't want to use global variables to access the results, they should be in StaticAnalysisContext instead now